messages.php

```diff
@@ -0,0 +1,48 @@
+<?php
+// Connect to the database
+$servername = "localhost";
+$username = "dev640";
+$password = "dev640password";
+$dbname = "dev640socialapp";
+$conn = new mysqli($servername, $username, $password, $dbname);

```

Comment:
No need to connect to the database here. We do database connection in header.php. You should require header.php in this page.